src/README.md

```diff
@@ -9,7 +9,39 @@ To build the docs locally, follow the steps mentioned below:
  6. ``cd ../src``
  7. ``make html``
 
-Documentation update PRs should only change the files in ``src``. 
+Documentation update PRs should only change the files in ``src``.
 
 Once the PR is merged to ``master``, you can find the auto-generated html files in ``docs`` directory.
 
+## conda-forge CODE OF CONDUCT ([https://conda-forge.org/docs/orga/governance.html#code-of-conduct])
```

Comment:
The best way to do a link is to pick some text to be the link text and then hide the URL. So this one above could be:

```conda-forge [Code of Conduct](https://conda-forge.org/docs/orga/governance.html#code-of-conduct)```